src/components/Layout/Header/NavBar/Native/BackupPassphraseModal/BackupPassphraseTest.tsx

```diff
@@ -0,0 +1,165 @@
+import { ArrowBackIcon } from '@chakra-ui/icons'
+import {
+  Button,
+  IconButton,
+  ModalBody,
+  ModalCloseButton,
+  ModalHeader,
+  Tag,
+  Wrap,
+} from '@chakra-ui/react'
+import * as native from '@shapeshiftoss/hdwallet-native'
+import { Vault } from '@shapeshiftoss/hdwallet-native-vault'
+import * as bip39 from 'bip39'
+import range from 'lodash/range'
+import shuffle from 'lodash/shuffle'
+import slice from 'lodash/slice'
+import uniq from 'lodash/uniq'
+import { useCallback, useEffect, useState } from 'react'
+import { useTranslate } from 'react-polyglot'
+import { useHistory } from 'react-router'
+import { SlideTransition } from 'components/SlideTransition'
+import { RawText, Text } from 'components/Text'
+
+import { BackupPassphraseRoutes } from './BackupPassphraseCommon'
+
+const Revocable = native.crypto.Isolation.Engines.Default.Revocable
+const revocable = native.crypto.Isolation.Engines.Default.revocable
+
+const TEST_COUNT_REQUIRED = 3
+
+const ordinalSuffix = (n: number) => {
```

Comment:
I know this is taken from a current implementation, but let's not spread this naming and make it clear this is a function and not an expression
```suggestion
const makeOrdinalSuffix = (n: number) => {
```